src/tooltip/tooltip.test.tsx

```diff
@@ -247,9 +247,12 @@ describe('Tooltip', () => {
         const button = screen.getByRole('button', { name: 'Click me' })
         userEvent.tab()
 
-        await waitFor(() => {
-            expect(button).toHaveAccessibleDescription('tooltip content here')
-        })
+        await waitFor(
+            () => {
+                expect(button).toHaveAccessibleDescription('tooltip content here')
+            },
+            { timeout: SHOW_DELAY + 10 },
```

Comment:
Default timeout is 1 second, so we basically just need one tick. This is a scalable solution, although each test would take 1 second (instead of 500ms before), so we can advance timers manually. Since we don't have that many tests, I think it is okay to keep it like that.